service/src/main/kotlin/fi/espoo/evaka/attachment/AttachmentQueries.kt

```diff
@@ -9,54 +9,45 @@ import fi.espoo.evaka.shared.AttachmentId
 import fi.espoo.evaka.shared.IncomeStatementId
 import fi.espoo.evaka.shared.auth.AuthenticatedUser
 import fi.espoo.evaka.shared.db.Database
-import fi.espoo.evaka.shared.db.bindNullable
+import fi.espoo.evaka.shared.db.mapColumn
 import fi.espoo.evaka.shared.domain.BadRequest
+import org.jdbi.v3.core.kotlin.bindKotlin
 import org.jdbi.v3.core.kotlin.mapTo
 import java.util.UUID
 
-sealed interface AttachTo
-
-data class AttachToApplication(val applicationId: ApplicationId) : AttachTo
-data class AttachToIncomeStatement(val incomeStatementId: IncomeStatementId) : AttachTo
-object AttachToNothing : AttachTo
+data class AttachmentParentColumn(
```

Comment:
Could this be `private`? Is there a need to expose this class? Or you could just move the class inside the the `insertAttachment` function if it's the only place where it's needed. Local classes don't need the `private` modified since they're only visible within the local scope (e.g. a function).

Reply:
Moved inside the function 👍